XGBoost/XGBoost (1).py

```diff
@@ -0,0 +1,90 @@
+# XGBoost
+
+import numpy as np
+import pandas as pd
+
+# Self defined Class for XGBoost
+class XGB:
+    def __init__(self, X, Y):
+        self.df = pd.DataFrame(columns=['age', 'fees'])

```

Comment:
You have used parameters for fixed data, please remove it and make it work for a general case.